include/crm/common/output_internal.h

```diff
@@ -874,6 +874,15 @@ void pcmk__output_and_clear_error(GError *error, pcmk__output_t *out);
         out_obj->end_list(out_obj);                 \
     }
 
+/*!
+ * \internal
+ *
+ * \brief Register basic formatted output messages
+ *
+ * \param[in,out] out The output functions structure.
+ */
+void pcmk__register_common_messages(pcmk__output_t *out);
+
```

Comment:
I lean to putting custom messages for tools in libpacemaker, since that's where the APIs that use them are / will be.

lib/pacemaker/pcmk_output.c is getting large, but we could start using pcmk_output_*.c files for various things

Reply:
I had avoided libpacemaker because attrd_updater is only linked against libcrmcommon.so at the moment.

Reply:
Eventually its functionality will be moved to libpacemaker, so it will have to link against it anyway

Reply:
Okay, I'll move it to pcmk_output.c and then we can do a followup PR that splits that file into more managable chunks.